packages/models-library/src/models_library/api_schemas_webserver/wallets.py

```diff
@@ -79,3 +78,58 @@ class PaymentTransaction(OutputSchema):
     )
     state_message: str = FieldNotRequired()
     invoice_url: HttpUrl = FieldNotRequired()
+
+
+PaymentMethodID: TypeAlias = IDStr
+
+
+class PaymentMethodInit(OutputSchema):
+    wallet_id: WalletID
+    payment_method_id: PaymentMethodID
+    payment_method_form_url: HttpUrl = Field(
+        ..., description="Link to external site that holds the payment submission form"
+    )
+
+    class Config(OutputSchema.Config):
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "examples": [
+                {
+                    "wallet_id": 1,
+                    "payment_method_id": "pm_0987654321",
+                    "payment_method_form_url": "https://example.com/payment-method/form",
+                }
+            ]
+        }
+
+
+class PaymentMethodGet(OutputSchema):
+    idr: PaymentMethodID
+    wallet_id: WalletID
+    card_holder_name: str
+    card_number_masked: str
+    card_type: str
+    expiration_month: int
+    expiration_year: int
+    street_address: str
+    zipcode: str
+    country: str
+    created: datetime
+
+    class Config(OutputSchema.Config):
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "examples": [
+                {
+                    "idr": "pm_1234567890",
+                    "wallet_id": 1,
+                    "card_holder_name": "John Doe",
+                    "card_number_masked": "**** **** **** 1234",
+                    "card_type": "Visa",
+                    "expiration_month": 10,
+                    "expiration_year": 2025,
```

Comment:
these are some standard fields? do we need address/zipcode/country?

Reply:
The address/zipcode and country I believe is for the billing address. I took them from amn example.
At the end of the day it depends on app-motion .. this can definitively change